media/css/firefox/accounts.scss

```diff
@@ -0,0 +1,326 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../pebbles/includes/lib';
+@import '../hubs/sections';
+@import '../quantum/common';
+
+/* -------------------------------------------------------------------------- */
+// Common elements
+
+main {
+    background: #003eaa;
+    background: url('/media/img/firefox/accounts/tail.png') top -350px center no-repeat,
+                linear-gradient(to bottom, #003eaa 0%, #00b0ff 100%);
+    color: #fff;
+}
+
+/* -------------------------------------------------------------------------- */
+// Top header logos
+
+.top-header {
+    text-align: center;
+
+    h1 {
+        @include font-size-level2;
+        margin: 40px 0;
+    }
+
+    .header-logos {
+        margin-bottom: 20px;
+
+        .mozilla {
+            display: none;
+        }
+    }
+
+    @media #{$mq-desktop} {
+
+        h1 {
+            margin: 40px 0 80px;
+        }
+
+        .header-logos {
+            @include clearfix;
+
+            .firefox {
+                float: left;
+            }
+
+            .mozilla {
+                display: block;
+                float: right;
+            }
+        }
+    }
+}
+
+html[dir="rtl"] .top-header {
+    @media #{$mq-desktop} {
+        .header-logos {
+
+            .firefox {
+                float: right;
+            }
+
+            .mozilla {
+                float: left;
+            }
+        }
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Main page content container
+
+.main-content {
+    @include clearfix;
+    margin-bottom: 100px;
+
+    @media #{$mq-desktop} {
+        min-height: 400px;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Main page copy
+
+.main-copy {
+    @include background-size(280px, 122px);
+    background-image: url('/media/img/firefox/accounts/sync-devices.png');
+    background-position: top center;
+    background-repeat: no-repeat;
+    margin: 0 auto;
+    max-width: 400px;
+    padding-top: 140px;
+    text-align: center;
+
+    .tagline {
+        @include font-size-level4;
+    }
+
+    ol {
+        list-style: decimal;
+        list-style-position: inside;
+
+        li {
+            margin-bottom: 10px;
+        }
+    }
+
+    @media #{$mq-desktop} {
+        @include background-size(350px, 153px);
+        background-position: top left;
```

Comment:
This should probably stay `top center` at/beyond `$mq-desktop` for states that do not show the form.

![screen shot 2018-02-20 at 11 33 24 am](https://user-images.githubusercontent.com/317532/36439270-fac73306-1631-11e8-96b4-374b155d04ff.png)

Reply:
Good catch 👍